<Type Name="AsynchronousOperationException" FullName="System.IdentityModel.Services.AsynchronousOperationException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="c97aca285e62ee41a2f1b2f95cc45e1bf505141e" />
    <Meta Name="ms.sourcegitcommit" Value="d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="fr-FR" />
    <Meta Name="ms.lasthandoff" Value="04/03/2018" />
    <Meta Name="ms.locfileid" Value="30417683" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class AsynchronousOperationException : Exception" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit AsynchronousOperationException extends System.Exception" />
  <TypeSignature Language="DocId" Value="T:System.IdentityModel.Services.AsynchronousOperationException" />
  <TypeSignature Language="VB.NET" Value="Public Class AsynchronousOperationException&#xA;Inherits Exception" />
  <TypeSignature Language="C++ CLI" Value="public ref class AsynchronousOperationException : Exception" />
  <AssemblyInfo>
    <AssemblyName>System.IdentityModel.Services</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Exception</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
      <span data-ttu-id="48e27-101">Exception levée lorsqu'une erreur se produit pendant une opération asynchrone.</span>
      <span class="sxs-lookup">
        <span data-stu-id="48e27-101">The exception that is thrown when an error occurs during an asynchronous operation.</span>
      </span>
    </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="48e27-102">Initialise une nouvelle instance de la classe <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-102">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="48e27-103">Initialise une nouvelle instance de la classe <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-103">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="48e27-104">Ce constructeur initialise la propriété <xref:System.Exception.Message*> de la nouvelle instance en lui affectant un message système qui décrit l'erreur.</span><span class="sxs-lookup"><span data-stu-id="48e27-104">This constructor initializes the <xref:System.Exception.Message*> property of the new instance to a system-supplied message that describes the error.</span></span> <span data-ttu-id="48e27-105">Ce message prend en compte la culture système en cours.</span><span class="sxs-lookup"><span data-stu-id="48e27-105">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="48e27-106">Le tableau suivant montre les valeurs initiales des propriétés de la nouvelle instance.</span><span class="sxs-lookup"><span data-stu-id="48e27-106">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="48e27-107">Property</span><span class="sxs-lookup"><span data-stu-id="48e27-107">Property</span></span>|<span data-ttu-id="48e27-108">Value</span><span class="sxs-lookup"><span data-stu-id="48e27-108">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="48e27-109">`null`.</span><span class="sxs-lookup"><span data-stu-id="48e27-109">`null`.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="48e27-110">Chaîne du message d'erreur localisé.</span><span class="sxs-lookup"><span data-stu-id="48e27-110">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="innerException">
          <span data-ttu-id="48e27-111">
            <see cref="T:System.Exception" /> qui constitue la cause de l'exception actuelle.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-111">The <see cref="T:System.Exception" /> that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="48e27-112">Si le paramètre <c>innerException</c> n’est pas <see langword="null" />, l’exception actuelle est levée dans un bloc <see langword="catch" /> qui gère l’exception interne.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-112">If the <c>innerException</c> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="48e27-113">Initialise une nouvelle instance de la classe <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> avec un message d'erreur fourni par le système et une référence à l'exception interne ayant provoqué cette exception.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-113">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with a system-supplied error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="48e27-114">Une exception levée en conséquence directe d'une exception précédente doit contenir une référence à celle-ci dans la propriété <xref:System.Exception.InnerException>.</span><span class="sxs-lookup"><span data-stu-id="48e27-114">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException> property.</span></span> <span data-ttu-id="48e27-115">Le <xref:System.Exception.InnerException> propriété retourne la même valeur que celle qui est passée au constructeur.</span><span class="sxs-lookup"><span data-stu-id="48e27-115">The <xref:System.Exception.InnerException> property returns the same value that is passed into the constructor.</span></span>  
  
 <span data-ttu-id="48e27-116">Le tableau suivant montre les valeurs initiales des propriétés de la nouvelle instance.</span><span class="sxs-lookup"><span data-stu-id="48e27-116">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="48e27-117">Property</span><span class="sxs-lookup"><span data-stu-id="48e27-117">Property</span></span>|<span data-ttu-id="48e27-118">Value</span><span class="sxs-lookup"><span data-stu-id="48e27-118">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="48e27-119">La référence à l'exception interne.</span><span class="sxs-lookup"><span data-stu-id="48e27-119">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="48e27-120">Un message système qui décrit l’erreur.</span><span class="sxs-lookup"><span data-stu-id="48e27-120">A system-supplied message that describes the error.</span></span> <span data-ttu-id="48e27-121">Ce message prend en compte la culture système en cours.</span><span class="sxs-lookup"><span data-stu-id="48e27-121">This message takes into account the current system culture.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(System::String ^ message);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="48e27-122">Message d'erreur qui explique la raison de l'exception.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-122">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="48e27-123">Initialise une nouvelle instance de la classe <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> avec un message d'erreur spécifié.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-123">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with a specified error message.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="48e27-124">Ce constructeur initialise la <xref:System.Exception.Message> propriété de l’exception à l’aide du `message` paramètre.</span><span class="sxs-lookup"><span data-stu-id="48e27-124">This constructor initializes the <xref:System.Exception.Message> property of the new exception using the `message` parameter.</span></span> <span data-ttu-id="48e27-125">Le contenu de `message` doit être compréhensible par les utilisateurs.</span><span class="sxs-lookup"><span data-stu-id="48e27-125">The content of `message` is intended to be understood by humans.</span></span> <span data-ttu-id="48e27-126">L'appelant de ce constructeur doit vérifier que cette chaîne a été localisée pour la culture en cours du système.</span><span class="sxs-lookup"><span data-stu-id="48e27-126">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span>  
  
 <span data-ttu-id="48e27-127">Le tableau suivant montre les valeurs initiales des propriétés de la nouvelle instance.</span><span class="sxs-lookup"><span data-stu-id="48e27-127">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="48e27-128">Property</span><span class="sxs-lookup"><span data-stu-id="48e27-128">Property</span></span>|<span data-ttu-id="48e27-129">Value</span><span class="sxs-lookup"><span data-stu-id="48e27-129">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="48e27-130">`null`.</span><span class="sxs-lookup"><span data-stu-id="48e27-130">`null`.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="48e27-131">Chaîne du message d'erreur.</span><span class="sxs-lookup"><span data-stu-id="48e27-131">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected AsynchronousOperationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; AsynchronousOperationException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info">
          <span data-ttu-id="48e27-132">Objet <see cref="T:System.Runtime.Serialization.SerializationInfo" /> qui contient les données sérialisées de l'objet.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-132">A <see cref="T:System.Runtime.Serialization.SerializationInfo" /> object that holds the serialized object data.</span>
          </span>
        </param>
        <param name="context">
          <span data-ttu-id="48e27-133">Objet <see cref="T:System.Runtime.Serialization.StreamingContext" /> qui contient les informations contextuelles sur la source ou la destination.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-133">A <see cref="T:System.Runtime.Serialization.StreamingContext" /> object that contains the contextual information about the source or destination.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="48e27-134">Initialise une nouvelle instance de la classe <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> avec des données sérialisées.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-134">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with serialized data.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="48e27-135">Ce constructeur est appelé lors de la désérialisation pour reconstituer l’objet exception transmis sur un flux.</span><span class="sxs-lookup"><span data-stu-id="48e27-135">This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.Services.AsynchronousOperationException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(System::String ^ message, Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel.Services</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="48e27-136">Message d'erreur qui explique la raison de l'exception.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-136">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="48e27-137">
            <see cref="T:System.Exception" /> qui constitue la cause de l'exception actuelle.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-137">The <see cref="T:System.Exception" /> that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="48e27-138">Si le paramètre <c>innerException</c> n’est pas <see langword="null" />, l’exception actuelle est levée dans un bloc <see langword="catch" /> qui gère l’exception interne.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-138">If the <c>innerException</c> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="48e27-139">Initialise une nouvelle instance de la classe <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> avec un message d'erreur spécifié et une référence à l'exception interne ayant provoqué cette exception.</span>
          <span class="sxs-lookup">
            <span data-stu-id="48e27-139">Initializes a new instance of the <see cref="T:System.IdentityModel.Services.AsynchronousOperationException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="48e27-140">Une exception levée en conséquence directe d'une exception précédente doit contenir une référence à celle-ci dans la propriété <xref:System.Exception.InnerException>.</span><span class="sxs-lookup"><span data-stu-id="48e27-140">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException> property.</span></span> <span data-ttu-id="48e27-141">Le <xref:System.Exception.InnerException> propriété retourne la même valeur que celle qui est passée au constructeur.</span><span class="sxs-lookup"><span data-stu-id="48e27-141">The <xref:System.Exception.InnerException> property returns the same value that is passed into the constructor.</span></span>  
  
 <span data-ttu-id="48e27-142">Le tableau suivant montre les valeurs initiales des propriétés de la nouvelle instance.</span><span class="sxs-lookup"><span data-stu-id="48e27-142">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="48e27-143">Property</span><span class="sxs-lookup"><span data-stu-id="48e27-143">Property</span></span>|<span data-ttu-id="48e27-144">Value</span><span class="sxs-lookup"><span data-stu-id="48e27-144">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException>|<span data-ttu-id="48e27-145">La référence à l'exception interne.</span><span class="sxs-lookup"><span data-stu-id="48e27-145">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message>|<span data-ttu-id="48e27-146">Chaîne du message d'erreur.</span><span class="sxs-lookup"><span data-stu-id="48e27-146">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>